src/generator/generator.ts

```diff
@@ -232,14 +232,19 @@ export class Generator {
     const headers: Headers = this.options.includePrivate
       ? {}
       : {'X-User-Ip': '0.0.0.0'};
+    const ignore = require('../../../ignore.json').ignore as string[];
     const res = await this.request<Schemas>({url: discoveryUrl, headers});
     const apis = res.data.items;
     const queue = new Q({concurrency: 10});
     console.log(`Generating ${apis.length} APIs...`);
     queue.addAll(
       apis.map((api: {[key: string]: string}) => {
         return async () => {
-          this.log('Generating API for %s...', api.id);
+          // look at ignore.json to find a list of APIs to ignore
+          if (ignore.indexOf(api.id) >= 0) {
+            return;
```

Comment:
Log message will be helpful. AFAIR the generator prints APIs it generates, so makes sense to print APIs it ignores as well.

Reply:
Good call!